docs/dictionary/function/matrixMultiply.lcdoc

```diff
@@ -45,11 +45,9 @@ second part is the column number. For example, the expression
 myArray[3,2] describes the element of myArray which is in the third row,
 second column.
 
-The number of rows in the <firstArray> must be the same as the number of
-columns in the <secondArray>, and the number of rows in the
-<secondArray> must match the number of columns in the <firstArray>.
-However, the number of rows in the <firstArray> and <secondArray> need
-not be the same.
+The number of columns in the <firstArray> must be the same as the number
```

Comment:
@livecodealex I think this is actually an engine bug that the condition isn't being checked. If you look at this line https://github.com/livecode/livecode/blob/3ad55491c4ef7d7b7f2d950bb36de8e56727caf4/engine/src/exec-array.cpp#L1215 you will see `MCMatrixEntry(p_b, k, j)` expects there to be an element in the second array where row index = column index from the first array.

Reply:
I think my comment above is wrong after looking at it again... sorry